adapter/ipListChecker/ipListChecker_test.go

```diff
@@ -22,44 +22,42 @@ import (
 
 	"gopkg.in/yaml.v2"
 
+	"time"
```

Comment:
we really need a better `goimports` that does these things. should not be a code review thing.